plugins/gui/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM build.sofi.com:5000/sofi-node-base
```

Comment:
This can't be built outside of the SoFi docker repo. Can you build a public node base container?